packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/billingAccount.routing.js

```diff
@@ -106,7 +106,8 @@ export default /* @ngInject */ ($stateProvider) => {
       currentTheme: /* @ngInject */ (softphoneService, billingAccountId) =>
         softphoneService
           .getSoftphoneCurrentGlobalTheme(billingAccountId)
-          .then(({ themeId }) => themeId),
+          .then(({ themeId }) => themeId)
+          .catch(() => {}),
```

Comment:
If the default behavior is to not fail on error, catching error should be managed directly in the service rather in each routing file